micro-ui/web/packages/modules/workbench/CHANGELOG.md

```
@@ -1,5 +1,8 @@
 # Changelog 
 
+## ["2.0.0-dev-03"] [16-Dec-2025]
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)